ecommerce-demo/dashboardPlugin/meta.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Should we rename `dashboardPlugin` → `dashboardPlugins` (in plural)? So that it makes more sense to add some more plugins to the folder in the future.

Also, with this logic, I think we should rename `meta.json` → `gd_forecasting.json` because we are defining  infos related to the forecasting plugin, aren't we?

Reply:
fixed